### PR TITLE
mrc-2737 Indicate Data Exploration mode in login screen

### PR DESCRIPTION
**Pull request by Nicolas-Dolan**
## Description

To test, observe "Naomi Data Exploration" title when redirecting from `/explore` but not `/` if not logged in. See [mockups](https://imperiallondon.sharepoint.com/:p:/r/sites/MRCGIDA-hint-dev/_layouts/15/doc2.aspx?sourcedoc=%7BB91CFC01-B73F-44E3-9A30-D8E9BA47F710%7D&file=Naomi%20Data%20Exploration%20Mode%20mockups%20v2.pptx&action=edit&mobileredirect=true&cid=b66e037c-583b-4830-a171-999bcc5cd123&PreviousSessionID=f5d20f11-2094-0a0d-3d98-81bbe44f5c37).

## Type of version change
Minor

## Checklist

- [x] I have incremented version number, or version needs no increment
- [x] The build passed successfully, or failed because of ADR tests

### CODE REVIEW

File: src/app/static/templates/login.ftl [hunk 1]
```diff
@@ -26,7 +26,11 @@
     </script>
 </head>
 <body>
-    <a href="https://www.unaids.org"><img src="public/images/unaids_logo.png" class="large-logo mx-auto mt-5"/></a>
+    <a href="https://www.unaids.org"><img src="public/images/unaids_logo.png" class="large-logo mx-auto mt-5 mb-4"/></a>
+    <#if RequestParameters?? && RequestParameters.redirectTo?? && RequestParameters.redirectTo?matches("explore")>
```
Comment (LekanAnni) [inline src/app/static/templates/login.ftl:30]:
```suggestion
    <#if ( RequestParameters.redirectTo )?? && RequestParameters.redirectTo?matches("explore")>
```

File: src/app/static/templates/login.ftl [hunk 2]
```diff
@@ -26,7 +26,11 @@
...
+        <h1 class="text-center"><strong>Naomi Data Exploration</strong></h1>
```
Comment (LekanAnni) [inline src/app/static/templates/login.ftl:31]:
```suggestion
        <h3 class="text-center"><strong>Naomi Data Exploration</strong></h3>
```

text is too large.

**Review by LekanAnni — CHANGES_REQUESTED** [review]
Looks good, work well.

See inline suggestions

File: src/app/static/templates/login.ftl (hunk 2 above)
Comment (Nicolas-Dolan) [reply src/app/static/templates/login.ftl:31]: I was going off the [mockups](https://imperiallondon.sharepoint.com/:p:/r/sites/MRCGIDA-hint-dev/_layouts/15/doc2.aspx?sourcedoc=%7BB91CFC01-B73F-44E3-9A30-D8E9BA47F710%7D&file=Naomi%20Data%20Exploration%20Mode%20mockups%20v2.pptx&action=edit&mobileredirect=true&cid=b66e037c-583b-4830-a171-999bcc5cd123&PreviousSessionID=f5d20f11-2094-0a0d-3d98-81bbe44f5c37) here

**Review by EmmaLRussell — COMMENTED** [review]
I think, rather than pulling out the RequestParameters in the template, it would be nicer to do that in the controller method, set a property in the model, and use that property in the template. That way it would be easier to use the `exploreApplicationTitle` in AppProperties rather than having that "Naomi Data Exploration" string living in two places.

I also think it would be fine to include "Naomi" as the the application title in that h1 for the main application - it seems like an omission that we don't already have that. 

So in that case. in LoginController.login you could have something like:
```
model["appTitle"] = if (request.getParameter("redirectTo") == "explore")
        {
            appProperties.exploreApplicationTitle
        }
        else
        {
            appProperties.applicationTitle
        }
```
(will need to add appProperties to the controller class).

Then in the template, would just need:
```
<h1 class="text-center"><strong>${appTitle}</strong></h1>
```

File: NEWS.md [hunk 3]
```diff
@@ -2,6 +2,10 @@
 
 * Indicate Data Exploration mode in login screen
 
+# hint 1.73.1
+
+* fix issues with displaying Input Time Series for DRC data
+
```
Comment (EmmaLRussell) [inline NEWS.md]: This needs to incremented above the latest master version

**Review by EmmaLRussell — APPROVED** [review]
Works for me.

**Review by LekanAnni — APPROVED** [review]
👍

**Pull request merged**